### PR TITLE
Fix80 check multiplicity handling

### LINKED ISSUE #80: Add test that neuron models heed event multiplicity
Events, in particular `SpikeEvent`s can include a multiplicity: one event object represents several spikes emitted by a single sender in a single time step. The `handle()` method of the receiving neuron must read out this information and apply it, typically by multiplying the weight with the multiplicity. I think this only makes sense for `SpikeEvent`s, for all other event types this should lead to an error.

We currently have no test checking that all models heed multiplicity information. Add a test for this.

### PR DESCRIPTION
**Pull request by heplesser**
This PR allows spike_generator to emit spikes with multiplicity, which is essential for testing multiplicity-handling in neurons. It also adds a test to check that (almost) all neurons handle multiplicity of incoming spikes properly. This should fix #80.

### CODE REVIEW

**Comment by heplesser** [comment]
@jougs @tammoippen Would you review?

File: models/spike_generator.h [hunk 1]
```diff
@@ -153,7 +153,9 @@ namespace nest
        stop           double - earliest time stamp of a potential spike event that is not emitted in
   ms
        spike_times    double array - spike-times in ms
-       spike_weights  double array - corrsponding spike-weights, the unit depends on the receiver
+       spike_weights  double array - corresponding spike-weights, the unit depends on the receiver
+       spike_multiplicities  int array - multiplicities of spikes, same length as spike_times;
+  mostly for debugging
```
Comment (tammoippen) [inline models/spike_generator.h]: The comment formatting seems off here.

**Comment by tammoippen** [comment]
Very nice. :+1:  from my side.

File: models/spike_generator.h [hunk 2]
```diff
@@ -148,15 +148,18 @@ namespace nest
...
-  ms
-       spike_times    double array - spike-times in ms
-       spike_weights  double array - corrsponding spike-weights, the unit depends on the receiver
-       precise_times        bool - see above
-       allow_offgrid_spikes bool - see above
-       shift_now_spikes     bool - see above
+       origin                double - Time origin for device timer in ms
+       start                 double - earliest possible time stamp of a spike to be emitted in ms
+       stop                  double - earliest time stamp of a potential spike event that is not
+  emitted in ms
```
Comment (abigailm) [inline models/spike_generator.h]: Are comments supposed to wrap like this, or should they be indented after the break?

**Comment by heplesser** [comment]
@abigailm I merged the formatting changes from the master branch now and fixed the comment formatting. I hope you are now happy with the PR.

**Pull request merged**